tools/wptrunner/wptrunner/executors/executorchrome.py

```diff
@@ -93,53 +93,41 @@ def setup(self):
         self.test_window = None
         self.reuse_window = self.parent.reuse_window
 
-    def close_test_window(self):
-        if self.test_window:
-            self._close_window(self.test_window)
-            self.test_window = None
-
     def close_old_windows(self):
         self.webdriver.actions.release()
         for handle in self.webdriver.handles:
             if handle not in {self.runner_handle, self.test_window}:
                 self._close_window(handle)
-        if not self.reuse_window:
-            self.close_test_window()
+        if not self.reuse_window and self.test_window:
+            self._close_window(self.test_window)
+            self.test_window = None
         self.webdriver.window_handle = self.runner_handle
-        # TODO(web-platform-tests/wpt#48078): Find a cross-platform way to clear
-        # cookies for all domains.
-        self.parent.cdp.execute_cdp_command("Network.clearBrowserCookies")
+        self._reset_browser_state()
         return self.runner_handle
 
-    def open_test_window(self, window_id):
-        if self.test_window:
-            # Try to reuse the existing test window by emulating the `about:blank`
-            # page with no history you would get with a new window.
-            try:
-                self.webdriver.window_handle = self.test_window
-                # Reset navigation history with Chrome DevTools Protocol:
-                # https://chromedevtools.github.io/devtools-protocol/tot/Page/#method-resetNavigationHistory
-                self.parent.cdp.execute_cdp_command("Page.resetNavigationHistory")
-                self.webdriver.url = "about:blank"
-                return
-            except error.NoSuchWindowException:
-                self.test_window = None
-        super().open_test_window(window_id)
-
-    def get_test_window(self, window_id, parent, timeout=5):
-        if self.test_window:
-            return self.test_window
-        # Poll the handles endpoint for the test window like the base WebDriver
-        # protocol part, but don't bother checking for the serialized
-        # WindowProxy (not supported by Chrome currently).
-        deadline = time.time() + timeout
-        while time.time() < deadline:
-            self.test_window = self._poll_handles_for_test_window(parent)
-            if self.test_window is not None:
-                assert self.test_window != parent
-                return self.test_window
-            time.sleep(0.03)
-        raise Exception("unable to find test window")
+    def _reset_browser_state(self):
```

Comment:
Let's have this function in its super class also, so that cross-browser methods(now empty) can be added there.